src/app/pages/entradas/ver-comprobante/ver-comprobante.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+@Component({
+  selector: 'ngx-ver-comprobante',
+  templateUrl: './ver-comprobante.component.html',
+  styleUrls: ['./ver-comprobante.component.scss'],
+})
+
+
+export class VerComprobanteComponent implements OnInit {
+  @Input() transaccion: any;
```

Comment:
Aprovechando lo comentado a la L2 de este archivo, y solo en caso que las constantes `MVTO_DB` y `MVTO_CR` no se puedan usar directamente en el .html
```suggestion
  m_db = MVTO_DB;
  m_cr = MVTO_CR;
  @Input() transaccion: any;
```
De confirmarse que `MVTO_DB` y `MVTO_CR` no se puedan usar directamente, probar adicional a lo anterior, que las variables anteriores también sean `const`